CaUWMET/src/demandsLogic/readInputDemands.py

```diff
@@ -0,0 +1,81 @@
+
+
+import os
+import numpy as np
+import pandas as pd
+from totalDemandsLogic import (
+    returnHydroRegion, reclassifyYearType,
+)
+
+# Input directories and filenames
+dirname = os.path.dirname(__file__)
+totalDemandsCSV = "../inputData/totalDemands.csv"
```

Comment:
You can take the "CSV" out of the variable names because they may not be CSV files - still trying to decide if we want to keep these as CSV or make a SQL database. Maybe "Input" would be best.